### DISCUSSION TITLE
Find latest version that is updated for all ...

### DISCUSSION THREAD

**Discussion by mjnagel** [renovatebot/renovate#12173] [Request Help] [answered]
Currently my renovate.json looks similar to this:
```json
{
    "baseBranches": ["main"],
    "configWarningReuseIssue": false,
    "dependencyDashboard": true,
    "dependencyDashboardAutoclose": true,
    "dependencyDashboardTitle": "Renovate: Upgrade Package Dependencies",
    "draftPR": true,
    "enabledManagers": ["helm-values","regex"],
    "ignorePaths": ["chart/charts/**"],
    "packageRules": [
        {
            "matchDatasources": ["docker"],
            "groupName": "images"
        }
    ],
    "regexManagers": [
        {
            "fileMatch": ["^chart/values\\.yaml$"],
            "matchStrings": [
                "image:\\s*(?<depName>.+):(?<currentValue>[\\d\\.]+)"
            ],
            "datasourceTemplate": "docker"
        }
    ],
    "separateMajorMinor": false
}
```

The match string regex matches 5+ docker images that are in my helm chart. When I run renovate against this config it works and opens issues/PRs to update to the latest BUT the image versions to update to are not always in sync. In this case I only want to update if all image versions are identical.

Ex:
- Current versions in repo: `img1:1.0`, `img2:1.0`, `img3:1.0`, ...
- Latest versions available on docker: `img1:1.1`, `img2:1.1`, `img3:1.2`, ...

With the current config I will get an issue/PR to update to `img1:1.1`, `img2:1.1`, `img3:1.2`, ...

Is there a way to make the issue/PR update to `1.1` only since that is the latest version that all images have?

**Comment by rarkins** [comment, marked as answer]
No, not currently

**Reply by mjnagel** [reply]
Do you happen to know if there is an open feature request for this? If not I'd be happy to open one.

**Reply by rarkins** [reply]
It's not an easy problem to solve, but you're welcome to raise a request. Please create a minimal reproduction repo first though,so that it's easier to understand. I.e. at least an example repo which exhibits the same problem when packages are updated.

BTW a workaround for it might simply be "stabilityDays=1" in your settings, so that it waits a day before raising a PR, which is likely enough time for all images to get the same version released.

**Reply by viceice** [reply]
https://github.com/renovatebot/renovate/issues/12133#issuecomment-943966256 looks related.

**Reply by HonkingGoose** [reply]
I think maybe the "roll up PR" concept can help out as well?

- https://github.com/renovatebot/renovate/issues/4404
